services/translateService.js

```diff
@@ -86,10 +62,11 @@ const setTranslateWithFallbackEngines = ({
     }
 
     if (!result) {
-      const enginesUsed = engines.join(", ");
+      const enginesUsed = engines.map((e) => e.name).join(", ");
 
       throw new Error(
-        `Error translating ${text} from ${from} to ${to} using ${enginesUsed}.\n\nPlease check that requested languages is supported using the command "languages" or check your internet connection and try again.\n\nFor more info check CLI help or open an issue at https://github.com/victor-heliomar/i18n-populator/issues/new`,
+        `Error translating ${text} from ${from} to ${to} using ${enginesUsed}.
+\nPlease check that requested languages is supported using the command "languages" or check your internet connection and try again.\n\nFor more info check CLI help or open an issue at https://github.com/victor-heliomar/i18n-populator/issues/new`,
```

Comment:
not sure how this change looks on the terminal, can you provide evidence or revert it as it was before?

```suggestion
        `Error translating ${text} from ${from} to ${to} using ${enginesUsed}.\n\nPlease check that requested languages is supported using the command "languages" or check your internet connection and try again.\n\nFor more info check CLI help or open an issue at https://github.com/victor-heliomar/i18n-populator/issues/new`,
```